paperdb/src/main/java/io/paperdb/Paper.java

```diff
@@ -45,6 +45,7 @@
      */
     public static void init(Context context) {
         mContext = context.getApplicationContext();
+        DependenciesProvider.getInstance().setApplicationContext(context);
```

Comment:
why didn't you pass mContext instead since it is application context

Reply:
Because existing code uses 2 constructors everywhere with and without context and we need to ensure we have context anyway for encryption so this would cause chain changes to make it work this way.

Reply:
Gotcha, I thought about different issue originally, will fix that.